lib/hpcbase.pm

```diff
@@ -12,17 +12,17 @@ sub enable_and_start {
 
 sub upload_service_log {
     my ($self, $service_name) = @_;
```

Comment:
Just cosmetics, but do you need `$self` here? why not just `my $service_name = shift` ?

Reply:
@kravciak  and this is why we need $self here https://openqa.suse.de/tests/1555850 . Test is failing because perl send $self as a first variable